chugbot/functions.php

```diff
@@ -139,7 +141,7 @@ function sendMail($address,
     if (array_key_exists("admin_email_from_name", $admin_data_row)) {
         $fromName = $admin_data_row["admin_email_from_name"];
     }
-    $mail->setFrom(ADMIN_EMAIL_USERNAME, $fromName);
+    $mail->setFrom("noreply@campramahchug.org", $fromName);
```

Comment:
The from email address needs to be formatted like an email address, so I just used this. If we kept using `ADMIN_EMAIL_USERNAME`, it would've been a random string of characters that AWS generated.

Reply:
Ah, that makes sense!  This looks good to me- great catch.  I spent 3 hours on this, and couldn't figure out why it wasn't working!  I suspect mailtrap.io (where I did initial debugging) is not strict about formatting.